### DISCUSSION TITLE
好きなひらがなは何ですか？/What is your favorite hiragana?

### DISCUSSION THREAD

**Discussion by kokonerumia** [NAIST-SE/AttractingContributors#1173] [2024 Attraction Contributions]
What is your favorite hiragana? Please feel free to write one character!
https://github.com/kokonerumia/favorite-hiragana/blob/main/README.md

**Comment by Marsall-k** [comment]
I have just a PR on your Repo.
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Comment by marshr00m** [comment]
I submitted a pull request. I would appreciate any additions to my list!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel
